### PR TITLE
NAS-125921 / 24.04 / Make sure middleware process respects CA bundle

### PR DESCRIPTION
**Pull request by sonicaj**

### CODE REVIEW

**Review by yocalebo — APPROVED** [review]

File: src/middlewared/middlewared/plugins/crypto_/__init__.py [hunk 1]
```diff
@@ -21,5 +21,10 @@ async def setup(middleware):
                 'Failed to set certificate for system.general plugin: %s', e, exc_info=True
             )
 
+    try:
+        await middleware.call('core.environ_update', {'REQUESTS_CA_BUNDLE': '/etc/ssl/certs/ca-certificates.crt'})
+    except Exception as e:
```
Comment (yocalebo) [inline src/middlewared/middlewared/plugins/crypto_/__init__.py:26]: Why not `failure=True`?

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/crypto_/__init__.py:26]: It does not really disrupt certificate setup process as that reflects to HTTPS always being available for NAS

Comment (themylogin) [reply src/middlewared/middlewared/plugins/crypto_/__init__.py:26]: Why can't this variable be put to the systemd unit file?

**Review by themylogin — APPROVED** [review]
Thank you!

**Pull request merged**